src/components/VDatePicker/mixins/date-table.js

```diff
@@ -51,9 +38,11 @@ export default {
     },
     dateGenTBody () {
       const children = []
-      const daysInMonth = this.normalizeDate(this.tableYear, this.tableMonth + 1, 0).getDate()
+      const daysInMonth = new Date(this.tableYear, this.tableMonth + 1, 0).getDate()
       let rows = []
-      const day = (this.normalizeDate(this.tableYear, this.tableMonth).getDay() - parseInt(this.firstDayOfWeek) + 7) % 7
+
+      // Use UTC time zone to get the position of the first day of the month relative to the first day of the week
+      const day = (new Date(`${this.tableYear}-${this.tableMonth + 1}-01 GMT+0`).getUTCDay() - parseInt(this.firstDayOfWeek) + 7) % 7
```

Comment:
This isn't very readable. I would suggest breaking it out into a separate function